js/src/controls/SearchControl.js

```diff
@@ -18,6 +18,7 @@ export class LeafletSearchControlModel extends control.LeafletControlModel {
       auto_collapse: false,
       zoom:10,
       animate_location:false,
+      found_style: {},
```

Comment:
The default value JavaScript side does not match the one Python side (this makes a difference when using embedded widgets)

Reply:
Done.